docs/sources/developers/http_api/playlist.md

```diff
@@ -32,7 +32,7 @@ Querystring Parameters:
 
 These parameters are used as querystring parameters.
 
-- **query** - Limit response to playlist having a name like this value.
+- **query** - Limit response to playlist having a name like this value. /!\ deprecated
```

Comment:
@papagian -- how are we noting APIs that we want to deprecate remove in the future?

In this case, we can really return the whole list of everything and search/filter on the frontend